### PR TITLE
[CIFuzz] Documentation update

### PR DESCRIPTION
**Pull request by Leo-Neat**
Makes features more obvious. Makes setting fuzz time more obvious.

### CODE REVIEW

File: docs/getting-started/continuous_integration.md [hunk 1]
```diff
@@ -34,6 +38,7 @@ You can integrate CIFuzz into your project using the following steps:
 file over from the OSS-Fuzz repository to the `workflows` directory.
 1. Change the `oss-fuzz-project-name` value in `main.yml` from `example` to the name of your OSS-Fuzz project. It is **very important** that you use your OSS-Fuzz project name which is case sensitive. This name
 is the name of your project's subdirectory in the [`projects`](https://github.com/google/oss-fuzz/tree/master/projects) directory of OSS-Fuzz.
+1. Set the value of `fuzz-seconds`. The longest time that the project maintainers are acceptable with should be used. This value should be at minimum 600 seconds and scale with a projects size.
```
Comment (inferno-chromium) [inline docs/getting-started/continuous_integration.md:41]: s/a projects size/project size

Comment (Leo-Neat) [reply docs/getting-started/continuous_integration.md]: Done.

File: docs/getting-started/continuous_integration.md [hunk 2]
```diff
@@ -14,11 +14,15 @@ fix bugs before they make it into your codebase
 
 ## How it works
 
-CIFuzz works by checking out a repository at the head of a pull request. The
-project's fuzz targets are built and run for
-a definite amount of time (default is 10 minutes). If a bug is found, the
-stack trace as well as the test case are made abailable for download.
- If a crash is not found the test passes with a green check.
+CIFuzz works by checking out a repository at the head of a pull request. For projects
+that support code coverage, fuzzers coverage is compared with PR diffs to determine
```
Comment (jonathanmetzman) [inline docs/getting-started/continuous_integration.md:18]: in "fuzzers coverage" drop "fuzzers".

File: docs/getting-started/continuous_integration.md [hunk 3]
```diff
@@ -14,11 +14,15 @@ fix bugs before they make it into your codebase
...
+which fuzzers should be used. For projects that do not support code coverage, all
```
Comment (jonathanmetzman) [inline docs/getting-started/continuous_integration.md:19]: "used" -> "run"

File: docs/getting-started/continuous_integration.md [hunk 4]
```diff
@@ -14,11 +14,15 @@ fix bugs before they make it into your codebase
...
+fuzzers are run for an even length of time. If no bugs are found and the allotted
```
Comment (jonathanmetzman) [inline docs/getting-started/continuous_integration.md:20]: "even length of time" -> "equal amount of time"

File: docs/getting-started/continuous_integration.md [hunk 5]
```diff
@@ -14,11 +14,15 @@ fix bugs before they make it into your codebase
...
+time is up (default is 10 minutes), the CI test passes with a green check. But
+if a bug is found, the bug is checked for reproducability and against
+old OSS-Fuzz builds to prevent the reporting of pre-existing bugs. If the bug is both
+new and reproducible, it is reported and the
```
Comment (jonathanmetzman) [inline docs/getting-started/continuous_integration.md:24]: maybe mention that CI fails.

**Pull request merged**